build_tarballs.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+CPE2CVE=cpe2cve
+CSV2CPE=csv2cpe
+NVDSYNC=nvdsync
+RPM2CPE=rpm2cpe
+NAME=nvdtools
+VERSION=1.0
```

Comment:
This is alright for now, but we'll need to get this from tags at some point. But it'll be a separate issue, methinks.